website/src/pages/edited-collections/chapter.page.tsx

```diff
@@ -39,7 +40,21 @@ const ChapterPage = (props: {
         <article className={dialog.visible ? css.leftMargin : util.fullWidth}>
           {/* If this chapter contains or is a Wordpress page, display the WP page contents. */}
           {wordpressId && chapter.slug ? (
+            <>
+            <header className={chapterStyle.docHeader}>
+            <Breadcrumbs aria-label="Breadcrumbs">
+            {chapter.breadcrumbs.map((crumb) => (
+              <Link href={`${collectionRoute(props.collectionSlug)}/${crumb.slug}`} key={crumb.slug}>
+                {crumb.name}
+              </Link>
+            )).concat(
+            <Link href={`${collectionRoute}/${chapter.slug}`} key={chapter.slug}>
```

Comment:
I think we have a chapterRoute function if that would be easier to use here! Otherwise, this looks good!